site/license-faq.html

```diff
@@ -573,7 +573,7 @@ <h3 id="32">I'm not using FusionAuth any more. How can I close my account and ca
               Once you have deleted all of your deployments and downgraded to Community Edition, your subscription will be canceled and you won’t be billed going forward.
             </p>
             <p>
-              If you have any questions regarding any of these steps, please contact sales@fusionauth.io. However, please note that we cannot do this for you.
+              If you would then like your user information completely removed please contact <a href="mailto:sales@fusionauth.io">sales@fusionauth.io</a>. Historical payment information must be maintained and does not contain any PII.
```

Comment:
I would point people to the contact form here rather than the sales email alias.

Reply:
updated in latest commit